### PR TITLE
add native helm repo url to services list

### PR DESCRIPTION
**Pull request by jsladerman**
also changes the help icon

### CODE REVIEW

File: assets/src/components/cd/services/ServicesColumns.tsx [hunk 1]
```diff
@@ -114,7 +114,11 @@ export const ColRepo = columnHelper.accessor(({ node }) => node, {
           <ColWithIcon
             truncateLeft
             icon={
-              helm ? getProviderIconUrl('byok', theme.mode) : <GitHubLogoIcon />
+              svc?.helm ? (
```
Comment (michaeljguarino) [inline assets/src/components/cd/services/ServicesColumns.tsx:117]: this is slightly wrong, we should really check if there's a helm url at all and show the helm-ish icon if so (you lose that info a bit by having one url var)

Comment (jsladerman) [reply assets/src/components/cd/services/ServicesColumns.tsx]: I changed it to just check if it has a git repo instead, I assume it's one or the other right?

Comment (michaeljguarino) [reply assets/src/components/cd/services/ServicesColumns.tsx]: Helm services can have git repos, so really the logic i gave is the best option

Comment (jsladerman) [reply assets/src/components/cd/services/ServicesColumns.tsx]: ah makes sense. fixed

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**